### PR TITLE
Add SOPs for RHOBS SLO alerts

### PR DESCRIPTION
**Pull request by ianbillett**
This PR adds one SOP / runbook for each of the SLO alerts we have now defined.

Signed-off-by: Ian Billett <ibillett@redhat.com>

### CODE REVIEW

File: docs/sop/observatorium.md [hunk 1]
```diff
@@ -89,7 +96,193 @@ Check targets are UP in app-sre Prometheus:
...
+
+Telemeter Server is returning a high-enough level of 5XX responses to write requests that we are depleting our SLO error budget.
+
+### Severity
+
+`critical`
+
+### Access Required
+
+- Console access to the cluster that runs Observatorium (Currently [telemeter-prod-01 OSD](https://console-openshift-console.apps.telemeter-prod.a5j2.p1.openshiftapps.com/project-details/all-namespaces)) and [app-sre-stage-0 OSD](https://console-openshift-console.apps.app-sre-stage-0.k3s7.p1.openshiftapps.com/project-details/all-namespaces))
```
Comment (jessicalins) [inline docs/sop/observatorium.md:115]:
```suggestion
- Console access to the cluster that runs Observatorium (Currently [telemeter-prod-01 OSD](https://console-openshift-console.apps.telemeter-prod.a5j2.p1.openshiftapps.com/project-details/all-namespaces) and [app-sre-stage-0 OSD](https://console-openshift-console.apps.app-sre-stage-0.k3s7.p1.openshiftapps.com/project-details/all-namespaces))
```
just noticed this extra `)` (was introduced [here](https://github.com/rhobs/configuration/pull/120)) :sweat_smile: 
if possible could you delete them from all the other `Access Required` section from this SOP? :pray:

File: docs/sop/observatorium.md [hunk 2]
```diff
@@ -89,7 +96,193 @@ Check targets are UP in app-sre Prometheus:
...
+* Check on the health of Telemeter Server
+  * Check the Telemeter [dashboard](https://grafana.app-sre.devshift.net/d/Tg-mH0rizaSJDKSADJ/telemeter)
+  * Check the logs of Telemeter Server pods.
+    * Telemeter Server should log any 5XX requests.
+* Check on the health of the API.
+  * Check the API [dashboard](https://grafana.app-sre.devshift.net/d/Tg-mH0rizaSJDKSADX/api)
+  * Check the logs on the API pods.
+* Check on the health of Thanos Receive.
+  * Check the Thanos Receive [dashboard](https://grafana.app-sre.devshift.net/d/916a852b00ccc5ed81056644718fa4fb/thanos-receive)
+  * Check the logs of the Thanos Receive pods.
```
Comment (jessicalins) [inline docs/sop/observatorium.md:130]: should we add a note to contact `#forum-observatorium` and `@observatorium-support` (specially for the `critical` alerts)

File: docs/sop/observatorium.md (hunk 1 above)
Comment (ianbillett) [reply docs/sop/observatorium.md:115]: Of course :)

File: docs/sop/observatorium.md (hunk 2 above)
Comment (ianbillett) [reply docs/sop/observatorium.md:130]: Yes good idea.

**Review by jessicalins — APPROVED** [review]
:avocado:

**Pull request merged**